src/main/java/frc/robot/Robot.java

```diff
@@ -99,9 +99,15 @@ public void robotInit() {
         SmartDashboard.putData("ring light on", led.ringLightOn());
         SmartDashboard.putData("ring light off", led.ringLightOff());
 
-        autoChooser.setDefaultOption("Nothing", new InstantCommand());
-        autoChooser.addOption("Pass the Line", drive.drivePastLine());
-        autoChooser.addOption("Shoot 3 Balls and Pass Line", shootAuto());
+        SmartDashboard.putData("Autonomous Test", drive.autonomousCommand("Slolom"));
+
+        autoChooser.setDefaultOption("Test", drive.autonomousCommand("Test"));
+        autoChooser.addOption("Slolom", drive.autonomousCommand("Slolom"));
+        autoChooser.addOption("Barrel", drive.autonomousCommand("Barrel"));
+        autoChooser.addOption("CollectBlue", drive.autonomousCommand("CollectBlue"));
+        autoChooser.addOption("CollectRed", drive.autonomousCommand("CollectRed"));
+        autoChooser.addOption("Bounce", drive.autonomousCommand("Bounce1").andThen(drive.autonomousCommand("Bounce2"))
```

Comment:
I think @msoucy commented elsewhere that you can pass multiple commands to and Then.